packages/grpc-native-core/index.d.ts

```diff
@@ -337,13 +343,13 @@ declare module "grpc" {
   /**
    * User provided method to handle server streaming methods on the server.
    */
-  type handleServerStreamingCall = (call: ServerWriteableStream) => void;
+  type handleServerStreamingCall<T> = (call: ServerWriteableStream<T>) => void;
 
   /**
    * A stream that the server can write to. Used for calls that are streaming
    * from the server side.
    */
-  export class ServerWriteableStream extends Writable {
+  export class ServerWriteableStream<T> extends Writable {
```

Comment:
The class `ServerWritableStream` is also responsible for sending responses, so I think it's a bit misleading to include only the request type in the type definition.